tools/packaging/common/envoy_bootstrap_v2.json

```diff
@@ -333,7 +333,10 @@
               ],
               "validation_context": {
                 "trusted_ca": {
-                  {{- if eq .pilot_cert_provider "kubernetes" }}
+                  {{- if .provided_cert }}
```

Comment:
nit: ProvCert is is `provisioned_cert` not `provided_cert` I believe

Reply:
modified to `provisioned_cert`